### ISSUE TITLE
RC Spot Check: John Travoltage 1.5.0-rc.2

### ISSUE THREAD

**Issue by jbphet** [phetsims/QA#200]
@KatieWoe and @ariel-phet, John Travoltage 1.5.0-rc.2 is ready for RC testing. Testing on 1.5.0-rc.1 was [completed](https://github.com/phetsims/QA/issues/196), several issues have been addressed, and this is the follow up test.  For the most part, a full test on all platforms is not necessary, just a verification of the issues and spot testing on key platforms.  The exception is that a full test should be performed on Internet Explorer, since a bug blocked testing of the previous RC.

Please document issues in https://github.com/phetsims/john-travoltage/issues and link to this issue.

Assigning @ariel-phet for prioritization.

<details>
<summary><b>General RC Test</b></summary>

<h3>What to Test</h3>

- One if the issues that was addressed was related to when electrons are added to the body based on foot movements (see issue list below).  Special care should be taken to make sure that nothing is broken with this behavior.
- The addition of sound was the primary purpose of this release.  While nothing has changed with respect to sound for this release, it's still good to pay close attention to it.

<h3>Issues to Verify</h3>

- [x] [Sim won't load on internet explorer](https://github.com/phetsims/john-travoltage/issues/314)
- [x] [Electrons not added during scrape sound](https://github.com/phetsims/john-travoltage/issues/313)
- [x] [LoL: Sound Continues After Pause](https://github.com/phetsims/john-travoltage/issues/309)
- [x] [Reset all positions foot differently](https://github.com/phetsims/john-travoltage/issues/312)
- [x] [Jaws performance and reading alerts](https://github.com/phetsims/john-travoltage/issues/310)
- [x] [Part of keyboard nav is untranslatable](https://github.com/phetsims/john-travoltage/issues/308)

These issues should have the "status:fixed-pending-testing" label. Check these issues off and change their status to fixed-awaiting-deploy if resolved. Otherwise, post a comment in the issue saying that it wasn't fixed.

<h3>Link(s)</h3>

- **[Simulation](https://phet-dev.colorado.edu/html/john-travoltage/1.5.0-rc.2/phet/john-travoltage_en_phet.html)**
- **[iFrame](https://phet-dev.colorado.edu/html/john-travoltage/1.5.0-rc.2/phet/john-travoltage_en_iframe_phet.html)**
- **[XHTML](https://phet-dev.colorado.edu/html/john-travoltage/1.5.0-rc.2/phet/xhtml/john-travoltage_all_phet.html)**
- **[Legends of Learning Harness](https://developers.legendsoflearning.com/public-harness/index.html?startPayload=%7B%22languageCode%22%3A%22en%22%7D&url=https%3A%2F%2Fphet-dev.colorado.edu%2Fhtml%2Fjohn-travoltage%2F1.5.0-rc.2%2Fphet%2Fjohn-travoltage_en_phet.html%3FlegendsOfLearning)**

<hr>

</details>

<details>
<summary><b>FAQs for QA Members</b></summary>

<br>
  
  

  <details>
  <summary><i>There are multiple tests in this issue... Which test should I do first?</i></summary> 
  
  Test in order! Test the first thing first, the second thing second, and so on.

  </details>

  <br>

  

  <details>
  <summary><i>How should I format my issue?</i></summary>
  
  Here's a template for making issues:

      <b>Test Device</b>

      blah

      <b>Operating System</b>

      blah

      <b>Browser</b>

      blah

      <b>Problem Description</b>

      blah

      <b>Steps to Reproduce</b>

      blah

      <b>Visuals</b>

      blah

      <details>
      <summary><b>Troubleshooting Information</b></summary>

      blah

      </details>

  </details>

  <br>

  

  <details>
  <summary><i>Who should I assign?</i></summary>
  
  We typically assign the developer who opened the issue in the QA repository.

  </details>
  
  <br>

  

  <details>
  <summary><i>My question isn't in here... What should I do?</i></summary>
  
  You should:

  1. Consult the [QA Book](https://github.com/phetsims/QA/blob/master/doc/qa-book.md).
  2. Google it.
  3. Ask Katie.
  4. Ask a developer.
  5. Google it again.
  6. Cry.

  </details>

<br>

<hr>

</details>

**Comment by ariel-phet** [comment]
@KatieWoe current top testing priority

**Comment by KatieWoe** [comment]
Devices to Test On (@jbphet let me know if you have any changes you want):
---
- [ ] Win 10 Chrome full
- [ ] Win 10 Edge full
- [ ] Win 10 Firefox full + Jaws + NVDA
---
- [ ] Mac OS 10.14 Chrome partial
- [ ] Mac OS 10.14 Safari full + Voice over
- [ ] Mac OS 10.14 Firefox partial
---
- [ ] Win 8.1 IE full
- [ ] Win 7 IE full
---
- [ ] Mac OS 10.13 Chrome partial
- [ ] Mac OS 10.13 Safari partial
- [ ] Mac OS 10.13 Firefox partial
---
- [ ] Chrome OS partial
---
- [ ] iOS 12 partial
- [ ] iOS 11 partial

**Comment by KatieWoe** [comment]
Just tested all the screen readers.  If time permits, another look from a second tester would be nice.

**Comment by jbphet** [comment]
The list of devices on which to test looks good to me.

**Comment by KatieWoe** [comment]
@jbphet @jessegreenberg QA is done

**Comment by jbphet** [comment]
I checked in with the phet-io folks to make sure that one of the issues linked above isn't seen as blocking the release.  Here is a record of the dialog:

>@jbphet [12:16 PM]
>Issue https://github.com/phetsims/scenery/issues/861 is referenced in the RC test ticket for John Travoltage, see https://github.com/phetsims/QA/issues/200.  I've read through it, and my impression is that I don't need to take any action related to this issue for the in-process release branch for JT, which is the 1.5 branch.  Do you phet-io gurus concur?

>@zepumph [12:24 PM]
>That's correct thanks for checking.

**Comment by jbphet** [comment]
Thanks for testing, I'll close this and follow up on the one issue that resulted.
